File: Programmers/Inho/Level01/최대공약수와 최소공배수.cs
using System;
public class Solution {
    public long[] solution(int n, int m) {
        long[] answer = new long[2] ;
        long nm = n*m;
        long lcm = n*m;
        long gcd;
        for(int i = 1 ; i<=m;i++){
            for(int j = 1; j<=n;j++){
                if(lcm < i*n)
                    continue;
                if(i*n==j*m)
                    lcm = i*n;
            }
        }
        Console.WriteLine("LCM은 {0}", lcm);
        gcd = nm/lcm;
        Console.WriteLine("GCD는 {0}",gcd);
        answer[0] = gcd;
        answer[1] = lcm;
        return answer;
    }
}

File: Programmers/Inho/C++/Level01/최대공약수와 최소공배수.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;

vector<int> solution(int n, int m) {
    vector<int> answer;
    int max = 0;
    for(int i = n; i<=n*m;i= i+n){
        for(int j = m; j<=n*m;j=j+m){
            if(i==j){
                max = i;
            }
        }
        if(max!=0)
            break;
    }
    int min = m*n/max;
    answer.push_back(min);
    answer.push_back(max);
    return answer;
}